tests/ui/feature-gates/feature-gate-gen_blocks.e2024.stderr

```diff
@@ -35,16 +35,16 @@ LL |     async gen {};
    = help: add `#![feature(gen_blocks)]` to the crate attributes to enable
 
 error[E0282]: type annotations needed
-  --> $DIR/feature-gate-gen_blocks.rs:5:9
+  --> $DIR/feature-gate-gen_blocks.rs:5:5
    |
 LL |     gen {};
```

Comment:
This is the span of the yield type variable. We're just using the whole span instead of the body's span. I don't think it matters here, because neither span is actually the most precise one to point at for "ambiguous yield type"... 🤷